src/scripts/Controller.js

```diff
@@ -169,6 +166,9 @@ export default class SitemapController {
 			'#sitemapFiles': {
 				change: this.readBlob,
 			},
+			'.copyText': {
```

Comment:
Some artifacts from previous pull request? You need to rebase to the latest master and only have file changes connected only with this pull request.